presto-native-execution/presto_cpp/main/QueryContextManager.cpp

```diff
@@ -51,14 +52,30 @@ folly::IOThreadPoolExecutor* spillExecutorPtr() {
 }
 
 namespace {
-std::string maybeRemoveNativePrefix(const std::string& name) {
-  static const std::string kNativePrefix = "native_";
-  const auto result =
-      ::strncmp(name.c_str(), kNativePrefix.c_str(), kNativePrefix.size());
-  if (result == 0) {
-    return name.substr(kNativePrefix.length());
-  }
-  return name;
+std::string fromPrestoToNativeConfig(const std::string& name) {
```

Comment:
Would you add a comment to explain how this function works, e.g. clarify that it returns 'name' as is if there is no mapping.